code/datums/components/surgery_initiator.dm

```diff
@@ -136,7 +136,7 @@
 	if(iscyborg(user))
 		close_tool = locate(/obj/item/cautery) in user.held_items
 		if(!close_tool)
-			patient.balloon_alert(user, "need a cautery in an inactive slot to stop the surgery!")
+			patient.balloon_alert(user, "need a [is_robotic ? "screwdriver": "cautery"] in an inactive slot to stop the surgery!")
```

Comment:
```suggestion
			patient.balloon_alert(user, "need a [is_robotic ? "screwdriver": "cautery"] in an inactive slot to stop the surgery!") // BUBBER EDIT
```

This can probably be brough up to tg but eh works too